R/Widget_TimeSeries.R

```diff
@@ -41,6 +41,48 @@ Widget_TimeSeries <- function(
     siteSelectLabelValue <- paste0("Highlighted ", siteSelectLabelValue, ": ")
   }
 
+  # rename results to account for rpt_* table refactor
+  # -- this is the data format expected by JS library {rbm-viz}
+  results <- results %>%
+    select(
+      "studyid" = "study_id",
+      "groupid" = "site_id",
+      "numerator",
+      "denominator",
+      "metric" = "kri_value",
+      "score" = "kri_score",
+      "flag" = "flag_value",
+      "gsm_analysis_date",
+      "snapshot_date"
```

Comment:
Hmm. I wonder if we should refactor the assess functions to use this naming. I guess this rename is happening in make_snapshot now ...

Reply:
yeah, going back and forth between these naming conventions is definitely a pain and will be hard to mainta. Let's discuss how to clean this up.